cmd/app/main.go

```diff
@@ -1,47 +1,22 @@
 package main
 
 import (
-	"context"
-	"github.com/gin-gonic/gin"
-	"github.com/oapi-codegen/gin-middleware"
 	"log"
 	"path/filepath"
-	"time"
+
+	"github.com/gin-gonic/gin"
+	ginmiddleware "github.com/oapi-codegen/gin-middleware"
 
 	"net/http"
+
 	"pagopa.it/pagopa-payment-wallet-helpdesk-service/cmd/app/api"
-	"pagopa.it/pagopa-payment-wallet-helpdesk-service/internal/cosmosdb"
-	"pagopa.it/pagopa-payment-wallet-helpdesk-service/internal/repository"
+	DB "pagopa.it/pagopa-payment-wallet-helpdesk-service/cmd/cosmosdb"
+	service "pagopa.it/pagopa-payment-wallet-helpdesk-service/cmd/service"
 )
 
 func main() {
-	mongoClient, err := cosmosdb.GetMongoClient()
-	if err != nil {
-		log.Panicf("Error connecting to MongoDB %v", err)
-	}
-	defer cosmosdb.CloseMongoClient()
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-	defer cancel()
-	paymentWalletRepository := repository.NewPaymentWalletRepository(mongoClient)
-	wallets, err := paymentWalletRepository.GetWalletsByUserID("6fd46190-f4bf-4a9f-9e70-fb98f235a449", ctx)
-	if err != nil {
-		log.Printf("Error searching for wallets %v", err)
-	} else {
-		totalWallets := len(wallets)
-		for idx, wallet := range wallets {
-			log.Printf("Wallet %d/%d -> %s", idx+1, totalWallets, wallet)
-			log.Printf("wallet id: [%s]", wallet.ID)
-			log.Printf("onboarding channel: [%s]", wallet.OnboardingChannel)
-			log.Printf("wallet status: [%s]", wallet.Status)
-			log.Printf("wallet user id: [%s]", wallet.UserID)
-			log.Printf("wallet detail type: [%s]", wallet.Details.Type)
-			if len(wallet.Applications) > 0 {
-				log.Printf("wallet application ID: [%s]", wallet.Applications[0].ID)
-				log.Printf("wallet application status: [%s]", wallet.Applications[0].Status)
-			}
-		}
-	}
-
+	cosmosdb, _ := DB.GetMongoClient()
```

Comment:
No check for error? :)
```suggestion
	cosmosdb, err := DB.GetMongoClient()
	if err != nil {
	    log.Fatalf("Unable to create Mongo client");
	}
```

Reply:
fixed with https://github.com/pagopa/pagopa-payment-wallet-helpdesk-service/pull/8/commits/1cbde9a29295174d7259e53597ad44815d13141f